packages/sui-bundler/README.md

```diff
@@ -164,6 +165,10 @@ This tool works with zero configuration out the box but you could use some confi
 **advice:** the alias option will not work for dev environment
 > The URL to the CDN **MUST** end with a slash `/`
 
+## OnlyHash
+
+In windows system filenames with `~` could produces errors. To avoid that you can use the flag `onlyHash` in your configurarion to go form names like `Home.123.js` to `123.js`. And that sould solve the issue.
```

Comment:
`could produces` -> `could produce` 
`configurarion` -> `configuration`
`sould` -> `should`